### ISSUE TITLE
the attack success rate on visual entailment.

### ISSUE THREAD

**Issue by carlos6889** [adversarial-for-goodness/Co-Attack#5]
Hi, thanks for your work for the adversarial attack against the VLP model. 
I have a question about the result of Co-attack attacks the ALBEF on VE task in this paper (Table 8).

First I run the VEFusionEval.py with adv=4, I get the adversarial acc=0.23. Then, I run the VEFusionEval.py with adv=0 and only
positive image-text pairs as input, I get the clean accuracy=0.81.
according to the ASR=clean accuracy-adversarial accuracy, the ASR=0.81-0.23=0.58, which is inconsistent with the paper reported.
<img width="817" alt="1680594072734" src="https://user-images.githubusercontent.com/129834302/229722738-be7b30a4-c54b-4b1b-aca0-7389df99d39f.png">

Can you analyze what the problem is?

**Comment by jiamingzhang94** [comment]
According to Section 4.2.1, for a fair comparison,  for the task that does not use cross-entropy, i.e., image-text retrieval,  our method and all baseline methods attack embedding-wise representation  as the target. For tasks that use cross-entropy, i.e. VE  and VG, our method and all baseline methods attack logit-wise  representation as the target. The VEFusionEval.py just provide the general method, i.e., attack embedding-wise representation.
